Index: b/champlainguifinet.py
===================================================================
--- a/champlainguifinet.py	2012-12-03 02:07:27.491500428 +0100
+++ b/champlainguifinet.py	2012-12-03 02:07:30.611389018 +0100
@@ -7,8 +7,6 @@
 
 import os
 import sys
-os.chdir(os.path.dirname(os.path.abspath(__file__)))
-sys.path.append('lib')
 
 from libcnml import Status
 
Index: b/guifinet-studio
===================================================================
--- a/guifinet-studio	2012-12-03 02:07:27.491500428 +0100
+++ b/guifinet-studio	2012-12-03 02:07:30.611389018 +0100
@@ -20,10 +20,11 @@
 
 from gi.repository import Gdk, Gtk
 
+import gzip
 import os
 import sys
-os.chdir(os.path.dirname(os.path.abspath(__file__)))
-sys.path.append('lib')
+os.chdir('/usr/share/guifinet-studio')
+sys.path.append('/usr/share/pyshared/guifinet-studio')
 
 from libcnml import CNMLParser, Status
 import pyGuifiAPI
@@ -599,13 +600,13 @@
 		dialog.set_website_label(_("Author's blog"))
 		dialog.set_license_type(Gtk.License.GPL_3_0)
 	
-		with open("AUTHORS") as f:
+		with open("/usr/share/doc/guifinet-studio/AUTHORS") as f:
 			authors_list = []
 			for line in f.readlines():
 				authors_list.append(line.strip())
 			dialog.set_authors(authors_list)
 		
-		with open("COPYING") as f:
+		with gzip.open("/usr/share/doc/guifinet-studio/COPYING.gz") as f:
 			dialog.set_license(f.read())
 		
 		dialog.run()
Index: b/utils.py
===================================================================
--- a/utils.py	2012-12-03 02:07:27.491500428 +0100
+++ b/utils.py	2012-12-03 02:07:30.611389018 +0100
@@ -21,9 +21,9 @@
 import os
 
 GUIFI_NET_WORLD_ZONE_ID = 3671
-I18N_APP_NAME = 'guifinetstudio'
+I18N_APP_NAME = 'guifinet-studio'
 APP_NAME = I18N_APP_NAME
-LOCALE_DIR = 'locale'
+LOCALE_DIR = '/usr/share/locale'
 
 import gettext
 gettext.bindtextdomain(APP_NAME, LOCALE_DIR)
Index: b/lib/libcnml/libcnml.py
===================================================================
--- a/lib/libcnml/libcnml.py	2012-12-03 02:07:03.192368245 +0100
+++ b/lib/libcnml/libcnml.py	2012-12-03 02:07:47.566783606 +0100
@@ -618,7 +618,7 @@
 
 	
 	def validateDTDLxml(self, tree):
-		dtdfile = 'tests/cnml.dtd'
+		dtdfile = 'cnml.dtd'
 		try:
 			with open(dtdfile, 'rb') as dtdfp:
 				dtd = etree.DTD(dtdfp)
